### DISCUSSION TITLE
Manage Roles

### DISCUSSION THREAD

**Discussion by xkenshin** [orchidsoftware/platform#1574] [Q&A]
Hi, Having trouble to manage role. 

1. From doc, i can see roles can added into PermissionServiceProvider . But doesnt specify where should we add this file (path)?
2. Once added to here, will the roles being added automatically in System/Roles?

Appreciate if anybody could help, thanks in advance

**Comment by noum** [comment]
@xkenshin `php artisan make:provider PermissionServiceProvider`
In app/Providers/PermissionServiceProvider.php, add the custom permissions.
In config/app.php's provider list, call PermissionServiceProvider class.
